src/game/spawn/player.rs

```diff
@@ -26,7 +26,7 @@ pub struct Player;
 fn spawn_player(
     _trigger: Trigger<SpawnPlayer>,
     mut commands: Commands,
-    images: Res<ImageAssets>,
+    image_map: Res<AssetMap<ImageKey>>,
```

Comment:
Given that we also use a sprite atlas in the context of a player, which is also a kind of "image map", this is some potentially very confusing wording. Consider finding a better name for `AssetMap` in general.

Reply:
@paul-hansen do you have an opinion here?

Reply:
How about `image_handles: Res<HandleMap<ImageKey>>`?

Reply:
Works for me

Reply:
No strong opinions here, since it's just a template so we can always rename to whatever we want.
I think making it clearer that it's handles not the actual assets might help some new people with the concepts slightly so I'm on board with that.

Reply:
Renamed to `HandleMap`.